edx_proctoring/views.py

```diff
@@ -979,6 +980,12 @@ def _get_relevant_attempt_per_user(self, attempts):
                 # Always return a verified attempt if it exists.
                 if attempt['status'] == ProctoredExamStudentAttemptStatus.verified:
                     onboarding_attempts_per_user[attempt['user_id']] = attempt
+                # Always return a non reset attempt if its ID is greater
+                if (
+                    existing_attempt['status'] == ProctoredExamStudentAttemptStatus.onboarding_reset
+                    and existing_attempt['id'] < attempt['id']
```

Comment:
Not sure if `id` is the best field to compare. I'm using `id` on the assumption that a higher ID would have been created more recently.

Reply:
assuming ID is the standard autoincrement field that's a pretty safe bet

there's probably some sequence of interleaved DB ops which would make it not true but it'd be pretty fiddly to set up for the same exam and user